packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.types.ts

```diff
@@ -1,12 +1,20 @@
-import { ITheme } from '../../Styling';
+import { ITheme, IStyle } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
 
 export interface IVerticalDividerProps {
   /**
    * Optional function to generate the class names for the divider for custom styling
    */
   getClassNames?: (theme: ITheme) => IVerticalDividerClassNames;
+  theme?: ITheme;
+  styles?: IStyleFunctionOrObject<IVerticalDividerPropsStyles, IVerticalDividerStyles>;
```

Comment:
A `styled` component also has the `className` prop that is attached to the root element (in this case `wrapper`).